graph/model/models_gen.go

```diff
@@ -2,27 +2,55 @@
 
 package model
 
+// A message is used to communicate conditions detected while executing a query on the server.
 type Message struct {
-	ID          string  `json:"id"`
-	Kind        *string `json:"kind"`
+	// Unique identifier. This can be used by clients to process the message independently of localization or gramatical changes.
+	ID string `json:"id"`
+	// Message type (information, warning, error).
+	// **Values:** information, warning, error.
```

Comment:
Do we need `// **Values:** information, warning, error.` here?

Reply:
Ok, how about I remove it from the previous line? So it will be.

```go
// Message type.
// **Values:** information, warning, error.
```

Reply:
Updated per previous, let me know if you meant something different.